OSVRUnreal/Plugins/OSVR/Source/OSVR/OSVR.Build.cs

```diff
@@ -52,5 +65,11 @@ public OSVR(TargetInfo Target)
 			}
 			);
 
+        if (Target.Platform == UnrealTargetPlatform.Win32 || Target.Platform == UnrealTargetPlatform.Win64)
+        {
+            AddThirdPartyPrivateStaticDependencies(Target, "OpenVR");

```

Comment:
These lines look SteamVR specific - we aren't requiring a specific render module, nor are we linking against OpenVR.